app/(pages)/[domain]/page.tsx

```
@@ -304,3 +340,113 @@ export default async function BusinessPage({ params }: PageProps) {
+        item: `https://${rootDomain}/${citySlug}`,
+      },
+    ],
+  };
+
+  const totalBusinesses = categories.reduce((sum, c) => sum + c.count, 0);
+
+  return (
+    <>
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(breadcrumbData) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)